test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -0,0 +1,232 @@
+module UnitTest {
+  use Reflection;
+  /*
+    Assert that a boolean condition is true.  If it is false, prints
+    'assert failed' and halts the program.
+
+    :arg test: the boolean condition
+    :type test: `bool`
+  */
+  pragma "insert line file info"
+  pragma "always propagate line file info"
+  proc assertTrue(test: bool) {
+    if !test then
+      __primitive("chpl_error", c"assert failed");
```

Comment:
I would suggest we implement an `AssertionError` class, mark all of these assertion functions as `throws`, and throw  the error when an assertion fails rather than call this primitive, which halts the program.

Reply:
I have implemented an `AssertionError` class.